gto-support-db/build.gradle

```diff
@@ -14,4 +14,5 @@ dependencies {
     compile project(':gto-support-core')
 
     testCompile project(':gto-support-testing')
+    androidTestCompile 'org.hamcrest:hamcrest-library:1.3'
```

Comment:
I'm assuming this was in MPDx? I'll work on getting a multidex PR together. Until then comment out the "com.facebook.stetho:stetho-js-rhino" dependency